### DISCUSSION TITLE
could you please add MathExpressionConverter for WPF?

### DISCUSSION THREAD

**Discussion by ali50m** [dotnet/wpf#7868] [Ideas]
Since we already have this in Xamarin/MAUI. I believe many of WPF developers need this for years.

https://learn.microsoft.com/en-us/xamarin/community-toolkit/converters/mathexpressionconverter
https://github.com/xamarin/XamarinCommunityToolkit/tree/main/src/CommunityToolkit/Xamarin.CommunityToolkit/Converters/MathExpressionConverter

https://learn.microsoft.com/en-us/dotnet/communitytoolkit/maui/converters/math-expression-converter
https://github.com/CommunityToolkit/Maui/tree/main/src/CommunityToolkit.Maui/Converters/MathExpressionConverter

**Comment by lindexi** [comment]
See https://github.com/Alex141/CalcBinding
